CHANGELOG.md

```diff
@@ -63,6 +63,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Updates storybook URL from /storybook-static/ to /reservoir/v0_26.
 - Updates all references to "design system" in the Storybook documentation to include "Reservoir" in the name of the design system.
 - Updates language on the `Welcome` page.
+- Updates to React 17.
 
 ### Removals
```

Comment:
Should we add the package removals here?

Reply:
Yes, that's a good idea.